packages/mui-base/src/SliderUnstyled/SliderUnstyled.js

```diff
@@ -234,7 +266,6 @@ const SliderUnstyled = React.forwardRef(function SliderUnstyled(props, ref) {
               <Thumb
                 data-index={index}
                 {...thumbProps}
-                {...getThumbProps()}
                 className={clsx(classes.thumb, thumbProps.className, {
                   [classes.active]: active === index,
                   [classes.focusVisible]: focusVisible === index,
```

Comment:
Take a look at what should be the `focusVisible` owner state for the thumb component.